src/material/button/testing/button-harness-filters.ts

```diff
@@ -8,8 +8,14 @@
 
 import {BaseHarnessFilters} from '@angular/cdk/testing';
 
+/** Possible button appearances. */
+export type ButtonVariant = 'basic' | 'raised' | 'flat' | 'icon' | 'stroked' | 'fab' | 'mini-fab';
```

Comment:
It's too bad we never came up with a name for the default =P I often don't know how to refer to the default button